### DISCUSSION TITLE
Option to show live console output, like sophia.ps1

### DISCUSSION THREAD

**Discussion by versedwildcat** [Sophia-Community/SophiApp#33] [Ideas]
Add an option that will show a live console log of the settings being applied, similar to the sophia.ps1 powershell window.

**Comment by farag2** [comment]
Is it really necessary due to this is a GUI app?

**Reply by TheBossMagnus** [reply]
> Is it really necessary due to this is a GUI app?
Maybe not, but a progress bar is needed and also a log with console output.

**Reply by farag2** [reply]
Log is created already. Where do you need another log to be created?))

**Reply by TheBossMagnus** [reply]
The log don't contain done change, maybe only for the issue #48

**Reply by farag2** [reply]
Could you explain which changes have to be reflected in the log? log points on the user's choice only: check or unchecked.

**Reply by TheBossMagnus** [reply]
[SophiApp-LENOVOIDEAPAD_3-132942426293472489.txt](https://github.com/Sophia-Community/SophiApp/files/8496405/SophiApp-LENOVOIDEAPAD_3-132942426293472489.txt)
I refer to something like option X is been enabled successfully.

**Comment by versedwildcat** [comment]
Hmm, perhaps not. But, it is definitely better (at least for me) than looking at a spinning circle for more 2 minutes.
It could also be useful for troubleshooting to see if anything ever goes wrong in real-time.
Perhaps as an advanced or debug option?
Overall it would be a nice feature to have but nothing really necessary.

**Comment by farag2** [comment]
We will consider adding a progress bar instead of a live console. In the end, this is a GUI app. :)
